src/pages/docs/packaging-applications/build-servers/codefresh-pipelines.md

```diff
@@ -0,0 +1,453 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2024-10-01
+modDate: 2024-10-01
```

Comment:
Hey @susanpann - do we need to hold this until October?

Reply:
Have confirmed there's no need to hold - have updated this!